arm-storage/2015-06-15/swagger/storage.json

```diff
@@ -319,20 +295,25 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts": {
-      "get": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/listKeys": {
+      "post": {
         "tags": [
           "StorageAccounts"
         ],
-        "operationId": "StorageAccounts_ListByResourceGroup",
-        "description": "Lists all the storage accounts available under the given resource group. Note that storage keys are not returned; use the ListKeys operation for this.",
+        "operationId": "StorageAccounts_ListKeys",
+        "description": "Lists the access keys for the specified storage account.",
         "parameters": [
           {
-            "name": "resourceGroupName",
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "name": "accountName",
```

Comment:
I am just keeping it consistent with the latest swagger 
https://github.com/Azure/azure-rest-api-specs/blob/master/arm-storage/2016-12-01/swagger/storage.json#L292